### DISCUSSION TITLE
Me ajudem

### DISCUSSION THREAD

**Discussion by Samos100** [community/community#88665] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I'm taking a website that is already ready on the web to edit some things. How do I get access to it? What program did you use for this edition? notepad++, VS Studio?
It's my first time doing this, please help!!!

**Comment by queenofcorgis** [comment]
Hi @Samos100 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
